lighthouse-core/test/audits/installable-manifest-test.js

```diff
@@ -35,12 +35,14 @@ describe('PWA: webapp install banner audit', () => {
   describe('basics', () => {
     it('fails if page had no manifest', () => {
       const artifacts = generateMockArtifacts();
+      artifacts.InstallabilityErrors.errors.push({errorId: 'no-manifest', errorArguments: []});
       artifacts.WebAppManifest = null;
       const context = generateMockAuditContext();
 
       return InstallableManifestAudit.audit(artifacts, context).then(result => {
         assert.strictEqual(result.score, 0);
-        assert.ok(result.explanation.includes('No manifest was fetched'), result.explanation);
+        const items = result.details.items;
+        assert.ok(items[0].reason.formattedDefault.includes('no manifest'));
```

Comment:
```suggestion
        expect(items[0].reason).toBeDisplayString(/no manifest/);
```

this is the "proper" way to test the localized version of the string (or `i18n.getFormatted(items[0].reason, 'en-US').includes('no manifest')`) Mostly the same in practice and in theory, but
- doesn't reach into the internal representation (allowing for easier i18n refactors in the future) and
- is testing the final string coming out of the i18n pipeline, so it's closer to what will be in the actual report (`formattedDefault` is the fallback in case that pipeline fails)

mainly I expect people will be copy/pasting from these tests in the future so it's good to do it the slightly better way.

Reply:
this is very clean, i like it 👀